Mathlib/Algebra/BigOperators/Group/Finset.lean

```diff
@@ -1178,6 +1178,21 @@ lemma mulSupport_prod (s : Finset ι) (f : ι → α → β) :
   simp only [mulSupport_subset_iff', Set.mem_iUnion, not_exists, nmem_mulSupport]
   exact fun x ↦ prod_eq_one
 
+@[to_additive]
```

Comment:
Could you add a docstring to this lemma (and to its additivized version)? I had to read it several times to understand what it means. Thanks!

Reply:
That's a good point. While I didn't have too much trouble understanding the idea, it *did* take me some time to understand why `PairwiseDisjoint` is needed, so commenting on that would be especially useful (especially since we already have [Finset.prod_map](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Algebra/BigOperators/Group/Finset.html#Finset.prod_map), where this assumption is not necessary, since `Finset.map` uses an embedding but `Finset.image` doesn't).